Dockerfile

```diff
@@ -0,0 +1,13 @@
+# nginx:mainline-alpine3.20-slim
+FROM nginx@sha256:e9293c9bedb0db866e7d2b69e58131db4c2478e6cd216cdd99b134830703983a
+
+COPY ./nginx.conf /etc/nginx/nginx.conf
+COPY ./angular/dist/angular-starter /var/www/html
+
+RUN chown -R nginx:nginx /var/cache/nginx && \
```

Comment:
1. Can you pls check whether all these operations are needed?
2. Try to pass multiple files to single `chown` command

Reply:
Those commands were used by me to solve the problem of running Nginx as non root user.
I decided to switch to the other option: pull an official unprivileged Nginx image. It seems more suiting for the task
Since now I found a simpler solution, chown command not needed anymore